backend/api/blogpost_content/migrations/0002_auto_20200521_2110.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.0.5 on 2020-05-21 21:10
```

Comment:
I don't really understand why a migration was generated for `blogpost_content` when I didn't touch it... Can anyone explain this? It does seem to work fine though 🙃